open-grant-proposals/rfp-filecoin-block-explorer.md

```diff
@@ -0,0 +1,140 @@
+user needs, we would mainly implement the following function:
+
+- Allow users to search the details of block, message and miner by blockCid, messageCid, address, or nickname.
+
+![image text](../img/FILscan_search.png)
+
+- Show the statistical data rendered in the view
+
+![image text](../img/FILscan_statistic.png)
+
+- Show recent transaction history in the form of curve diagram
+
+![image text](../img/FILscan_transaction_history.png)
+
+- Show latest blocks and transactions 
+
+![image text](../img/FILscan_latest_blocks.png)
+ 
+- Show the list of message
+
+![image text](../img/FILscan_message_list.png)
+
+- Show the list of miner
+
+![image text](../img/FILscan_miner_list.png)
+ 
+- Show the details of block
+
+![image text](../img/FILscan_block_details.png)
+
+- Show the details of message
+
+![image text](../img/FILscan_message_details.png)
+
+- Show the details of miner
+
+![image text](../img/FILscan_miner_details.png)
+
+- Real-time updates to the date got from the chain
+
+You will be able to see the block confirmation process in real time without wasting time waiting and constantly 
+refreshing. All content from blocks, transactions, balances, and token transfers will be displayed in real time. 
+Also you can directly on your account page to view the transaction records and balances in real time once the 
+block is updated.
+
+Our block explorer website would be hosted on github, you can look over our github repository to 
+read the source code. 
+
+# Our Team
+
+## Team Members
+
+- BiSheng Mao : Lead of FILscan
+- WenYi Li : Technical Lead of FILscan
+- ZhaoMing Wang : Project Manager
+- Huan Shen : Filecoin Engineer and Researcher
+- Tao OuYang : UI Designer
+- Qi Zhang : UI & UE
+
+## Team Member LinkedIn Profiles
+
+- https://github.com/maobisheng (personal)
+- https://github.com/XQYCHJ (personal)
+- https://www.linkedin.com/feed/?trk=onboarding-landing
+
+## Relevant Experience
+
+IPFSUNION is a team focused on the global ecological environment construction of IPFS and Filecoin. We have been 
+focusing on Filecoin's mining testing, spec studies, problem submission, and code contributions. In the near future, 
+our goal is to become a major player in Web3.0, providing users with valuable storage and retrieval services. We will
+make different layouts according to the development stage of filecoin. 
+
+# Milestones & Funding
+
+Total Funding Amount: $1500
+
+Milestone: 
+
+| Milestone No. | Milestone Description | Funding | Estimated Timeframe |
+| --- | --- | --- | --- |
+| 1 | High-fidelity design mockups, running staging site with some UI elements implemented | $400 | 2 weeks |
+| 2 | Completed implementation of block explorer | $800 | 4 weeks |
+| 3 | All project deliverables are completed and added to the [Filecoin Shipyard](https://github.com/filecoin-shipyard) (website, documentation, codebase) | $300 | 2 weeks |
+
```

Comment:
Hi @IPFSUNION,

Some questions about your dev grant.

1. The amount requested seems really small. What specifically do you need a dev grant for?

2. above it says "the version V1.0 in March next year" - to confirm, this means March 2020 right?

Reply:
Some questions about your dev grant proposal.
The amount requested seems really small. What specifically do you need a dev grant for?
——Before we apply the devgrant, we have already started the filscan project in the Oct of 2019, we just want some bonus for it. 

above it says "the version V1.0 in March next year" - to confirm, this means March 2020 right?
——Yes, we have almost finished it when we submit the devgrant last year, and we will open the url "filscan.in" in the coming two weeks.

Reply:
@maobisheng - filscan.io was dev granted to a group in wave 1. Is it possible for this to have a different name? e.g. filscout?

Reply:
> @maobisheng - filscan.io was dev granted to a group in wave 1. Is it possible for this to have a different name? e.g. filscout?

No problem.